nginx/servers_config.sls

```diff
@@ -100,17 +104,19 @@ nginx_server_available_dir:
     file.absent:
         - name: {{ server_curpath(server) }}
 {% else %}
-{% if settings.config != None and settings.enabled == True %}
-{% if 'source_path' in settings.config %}
-{% set source_path = settings.config.source_path %}
-{% else %}
-{% set source_path = 'salt://nginx/files/server.conf' %}
-{% endif %}
+{% if settings.enabled == True %}
```

Comment:
I haven't looked into this but is there a reason you removed the `settings.config != None`?

Reply:
I removed this to be able to not specify a config object so that flat files can be retrieved by TOFS without this param set empty